pom.xml

```diff
@@ -161,37 +152,32 @@
         <dependency>
             <groupId>org.jenkinsci.plugins</groupId>
             <artifactId>pipeline-model-definition</artifactId>
-            <version>1.3.2</version>
+            <version>1.7.1</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
```

Comment:
No just delete altogether. One advantage of the BOM is that you do not need to explicitly specify these lower-level dependencies at all.